src/main/cljc/jarl/builtins/crypto.cljc

```diff
@@ -0,0 +1,95 @@
+(ns jarl.builtins.crypto
+  (:require [jarl.encoding.hex :as hex]
+            [jarl.builtins.utils :refer [str->bytes]]
+            #?(:clj  [jarl.encoding.base64 :as base64])
+            #?(:clj  [clojure.string :as str])
+            #?(:cljs [goog.crypt :as crypt]))
```

Comment:
Is this a transitive dependency from somewhere?

Reply:
ClojureScript is compiled using the Google Closure (yeah, quite a clash of names!) compiler, which provides the Closure library for "free". Those libraries are hence guaranteed to be present in any cljs project: https://www.clojurescript.org/about/closure